### DISCUSSION TITLE
database error after npx prisma migrate dev name init

### DISCUSSION THREAD

**Discussion by pedropankaj** [planetscale/discussion#80] [Bug Reports] [answered]
Error: Database error
Error querying the database: Server error: `ERROR 70100 (1317): held locks released: vttablet: rpc error: code = Aborted desc = transaction 1629464864154001725: ended at 2021-08-20 23:14:08.316 UTC (exceeded timeout: 20s) (CallerID: unsecure_grpc_client)'
   0: migration_core::api::ApplyMigrations
             at migration-engine/core/src/api.rs:69

plz anybody help

**Comment by jgreet** [comment, marked as answer]
Hi @pedropankaj, if you're still hitting that timeout could you share the migration that you're attempting to run? 

You could start a support ticket with the migration if you'd rather keep it private: https://planetscale.com/support
